cmd/ogamed/main.go

```diff
@@ -195,5 +195,9 @@ func start(c *cli.Context) error {
 	e.POST("/bot/planets/:planetID/cancel-research", ogame.CancelResearchHandler)
 	e.GET("/bot/planets/:planetID/resources", ogame.GetResourcesHandler)
 	e.POST("/bot/planets/:planetID/send-fleet", ogame.SendFleetHandler)
+
+	// IPM
+	e.GET("/bot/planets/:planetID/send-ipm/:ipmamount/:galaxy/:system/:position/:type", ogame.SendIPMHandler)
```

Comment:
This should be a POST endpoint.
and "ipmAmount"/"galaxy"/"system"/"position"/"type" should be part of the body.